src/Dapi/Types/ACHGetTransfer.cs

```diff
@@ -0,0 +1,13 @@
+namespace Dapi.Types {
+    public class ACHGetTransfer {
```

Comment:
Rename it to `ACHPullTransferInfo`.

Reply:
a0a7d568e2a48f45f3142ab56ff17bb5465db72b